Representable/representable
# State map page optimization added

Changes:
* state map page optimization added
* addresses removed

To test:
* Run python manage.py runserver
* python manage.py makemigrations
* python manage.py migrate
* create state objects by going to the admin page
* create new entries for state objects that already exist
* go to the state map page /map/<state abbreviation>
* check that the most recent entries you made that have state objects as foreign keys show up

![image](https://user-images.githubusercontent.com/15281033/103395039-120c2f80-4afa-11eb-9876-8d164ee63c3d.png)
